db/migrate/20160807063212_add_portrait_to_person.rb

```diff
@@ -0,0 +1,5 @@
+class AddPortraitToPerson < ActiveRecord::Migration
+  def change
+    add_column :people, :portrait, :string
+  end
+end
```

Comment:
Whoops, this is a change method, not an up. Change means rails knows how to do the up and the down.